src/Language/PureScript/Sugar/ObjectWildcards.hs

```diff
@@ -24,6 +27,9 @@ desugarObjectConstructors
   -> m Module
 desugarObjectConstructors (Module ss coms mn ds exts) = Module ss coms mn <$> mapM desugarDecl ds <*> pure exts
 
+data PathNode = Leaf Expr | Branch PathTree
```

Comment:
I know this isn't exported, but could you add a short comment explaining what it's for anyway? Thanks.